README.md

```diff
@@ -9,3 +9,37 @@ Run as follows:
 ```
 $ cargo build --release && cargo run --release --bin vtable_bench
 ```
+
+
+# Why this is worth measuring
+
+Trait objects' vtables are passed around as fat pointers. If they're used
+rarely, this seems unlikely to make much performance difference either way.
+However, if we have lots of trait objects, and if we're expecting to have a
+reasonably large number of trait objects, and for many of those to be multiply
+aliased, it's not obvious if fat pointers are a better trade-off vs. storing the
+vtable pointer in the object itself.
```

Comment:
Does this insight relate to the cost of passing around a fat pointer, vs a normal pointer? i.e. pass a pointer by value amounts to copying. And copying two words will always take twice as long as copying one.

Reply:
Yes, but there are always two words *somewhere*: the pointer to the object and the vtable pointer. The interesting question is where the best place to store them is (as a fat pointer; or to stuff the vtable into the object itself).

Reply:
Understood, but I only now understood why passing them around frequently can impact performance. When the vtable is inside, you only copy one word.

I'll leave it up to you to decide if you want to expand the discussion to mention that.

Reply:
Point taken. Let me update it a bit.